draft/2023-02-22-this-week-in-rust.md

```diff
@@ -39,6 +39,9 @@ and just ask the editors to select the category.
 
 ### Rust Walkthroughs
 
+- [Build a Apache Kafka Producer/Consumer Application in Rust](https://dev.to/schultyy/how-to-build-a-simple-kafka-producerconsumer-application-in-rust-3pl4)
```

Comment:
Thank you for submitting this. We have already included this in a prior PR, but will keep it in this PR to update the desired title.